frontend/src/components/Themes/ThemesTable.columns.ts

```diff
@@ -0,0 +1,15 @@
+import { type ColumnDef, createColumnHelper } from "@tanstack/react-table"
+import type { ThemePublic } from "../../client"
+
+const columnHelper = createColumnHelper<ThemePublic>()
+
+export const themesColumns: ColumnDef<ThemePublic, string>[] = [
+  columnHelper.accessor("name", {
+    header: "Text",
+    cell: (info) => info.getValue(),
+  }),
+  columnHelper.accessor("description", {
+    header: "Created At",
```

Comment:
Should this header value be "description"?

![image](https://github.com/user-attachments/assets/230c3f42-2457-4cf7-92b8-f80d5f91079d)

Reply:
yes, indeed, missed that, corrected